automatic/apache-httpd/apache-httpd.nuspec

```diff
@@ -35,13 +35,13 @@ Example: `choco install apache-httpd --params '"/installLocation:C:\HTTPD /port:
 
 ## Notes
 
-- This package will install the latest Apache binaries provided at Apache Haus (http://www.apachehaus.com/) with OpenSSL 1.1.1g, brotli 1.0.7, nghttp 1.41.0, Zlib 1.2.10, PCRE 8.44, APR 1.7.0, APR-Util 1.6.1.
+- This package will install the latest Apache binaries provided at Apache Haus (http://www.apachehaus.com/) with OpenSSL 1.1.1p, brotli 1.0.9, nghttp 1.47.0, Zlib 1.2.12, PCRE2 10.40
```

Comment:
In the future, please do not update the description in the `.nuspec` file as it will be automatically imported from the `Readme.md` file.